src/metabase/api/log.clj

```diff
@@ -0,0 +1,12 @@
+(ns metabase.api.log
+  "`/api/log` endpoints."
+  (:require [compojure.core :refer [GET POST DELETE PUT]]
+            [metabase.api.common :refer [defendpoint define-routes]]
+            [metabase.logger :as logger]))
+
+(defendpoint GET "/"
```

Comment:
I'm thinking we could put this in the `api.util` namespace instead since it's just a single function and not all that likely to be more.